voltage/message.py

```diff
@@ -50,14 +50,19 @@ class MessageMasquerade(NamedTuple):
         The name of the masquerade.
     avatar: Optional[:class:`str`]
         The url to the masquerade avatar.
+    colour: Optional[:class:`str`]
+        CSS-compatible color of the username
```

Comment:
can we change this to the british version? also add `color` as an alias to `colour` like in embeds

Reply:
Done!